README.adoc

```diff
@@ -541,3 +541,7 @@ spec:
       mode: DISABLE
 ...
 ----
+
+== Stargazers over time
+
+[![Stargazers over time](https://starchart.cc/kiali/kiali.svg?variant=adaptive)](https://starchart.cc/kiali/kiali)
```

Comment:
Are you sure the syntax is correct? When I visit your branch I see the text, not the chart.

Reply:
I think so...

It's a copy paste from code generated in the website. Check

https://starchart.cc/Kiali/kiali